project-fob/Models/User.cs

```diff
@@ -29,6 +29,10 @@ public override bool Equals(object obj)
             {
                 return Equals(paramater);
             }
+            if (obj is String parameter)
```

Comment:
This `User.Equals` method looks a bit dangerous, you can compare it either another User object or a string - but you can't compare a string to it. e.g. if you have

```
	var user = new User
	{
		Id = 1,
		UserId = "Hello"
	};
	
	user.Equals("Hello"); // will be true
	"Hello".Equals(user); // will be false
```

and that isn't the way equals is supposed to work (i.e. if `x == y` is `true`, then `y == x` should also be `true`)

I couldn't spot where the string version of Equals is used, where is it?